servicecontrol/sequence-contains-no-matching-element.md

```diff
@@ -0,0 +1,50 @@
+---
+title: ServiceControl - InvalidOperationException: Sequence contains no matching element
+summary: ServiceControl Management Utility shows System.InvalidOperationException: Sequence contains no matching element
+reviewed: 2024-06-06
+---
+
+## Symptoms
+
+The following error is shown when launching the ServiceControl Management Utility (SCMU):

```

Comment:
Would be worth mentioning that this error was also occuring during the instance update process when upgrading 5.x to anything.